src/style.css

```diff
@@ -0,0 +1,131 @@
+.page {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+.page__content {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 198px;
+  height: 406px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__images {
+  margin: 32px 19px 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0 16px 4px;
```

Comment:
it`s optional, but you could set all padding for selector class and just add margin-top or -bottom for the next elements